JASP-Engine/JASP/po/R-nl.po

```diff
@@ -5912,7 +5912,7 @@ msgid "All effects are specified as nuisance"
 msgstr "Alle effecten zijn gespecificeerd als nuisance"
 
 msgid "There are nonpositive weights"
-msgstr "Er zijn non-positieve gewichten"
+msgstr "Er zijn niet-positieve gewichten"
```

Comment:
ergens eerder was het "niet positief" en nu is het "niet-positief". Dat is inconsequent...

Reply:
In geval van twijfel: met streepje.